core/lib/dal/src/blocks_dal.rs

```diff
@@ -2334,6 +2355,84 @@ impl BlocksDal<'_, '_> {
 
         Ok(results.into_iter().map(L::from).collect())
     }
+
+    pub async fn get_last_l2_block_bloom(&mut self) -> DalResult<Option<Bloom>> {
```

Comment:
Will return values be used somewhere in the future? Right now, the only caller just checks whether the return value is `Some(_)`; so this method could be refactored to `has_l2_blocks_with_bloom() -> DalResult<bool>`.